js/sign/src/utils/constants.ts

```diff
@@ -1,4 +1,6 @@
 export const ED25519_PK_SIGNATURE_ATTRIBUTE_NAME = 'ed25519PublicKey';
+export const ECDSA_P256_SHA256_PK_SIGNATURE_ATTRIBUTE_NAME =
```

Comment:
Would it make sense to combine these into a map and to be used with the SignatureType enum values as keys?

Reply:
Done!